docs/index.html

```diff
@@ -26,8 +26,8 @@ <h2>Start here</h2>
                                     <img class="ext-arrow-smaller" src="/assets/images/ext-arrow-black.svg" alt="">
                                 </a>
                             </li>
-                            <li><a href="/insomnia/send-your-first-request">Send your first request</a>.</li>
-                            <li><a href="/insomnia/import-export-data">Import your collection or document</a> from another API client.</li>
+                            <li><a href="/insomnia/get-started/collections/send-your-first-request">Send your first request</a>.</li>
+                            <li><a href="/insomnia/get-started/mock-server/import-export-data">Import your collection or document</a> from another API client.</li>
```

Comment:
@filfreire Sorry for not getting to this sooner! I tested both links locally and they worked. Should we also add all the other links that were added to the Get Started section nav here as well? https://github.com/Kong/insomnia-docs/pull/222/files#diff-97f574942d983ddf2cd64ab43db44b1a876cbd937ae460ea307b91763580cabc